loopout.go

```diff
@@ -844,6 +850,20 @@ func (s *loopOutSwap) waitForHtlcSpendConfirmed(globalCtx context.Context,
 		// timer.
 		case notification := <-s.blockEpochChan:
 			s.height = notification.(int32)
+
+			// Check if the on-chain HTLC's CLTV has been reached.
+			// If so, we will time out the swap because it's no
+			// longer safe for us to reveal the preimage.
+			if s.height >= s.CltvExpiry {
```

Comment:
Not that this situation is likely to happen due to timing but how about some buffer here? Like expiry height minus the final confirmation target so that you still have some time to sweep without the timeout path being active?

Reply:
Something like [this](https://github.com/lightninglabs/loop/blob/master/loopout.go#L647)?

Reply:
Yeah exactly